schemas/context/media-timed.schema.json

```diff
@@ -109,6 +129,16 @@
           "title": "Total pause duration",
           "$ref": "https://ns.adobe.com/xdm/data/measure",
           "description": "Describes the duration in seconds in which playback was paused by the user."
+        },
+        "xdm:idp": {
+          "title": "MVPD Identifier",
```

Comment:
Can you spell out MVPD and/or a link to the spec?

Reply:
Added a link to the spec and spelled it out in the description.